_posts/2020-02-01-reviving-sandstorm.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: "Reviving Sandstorm"
+author: Ian Denhardt
+authorUrl: https://github.com/zenhack
+---
+
+It's no secret that development on Sandstorm has been pretty sparse
```

Comment:
This looks great!

The only thing I'd say is maybe there should be a sentence or two added to the beginning that does two things:

1. Introduce yourself. All the posts in the last three years were written by me, and people might not notice the new authorship if it isn't pointed out.
2. Give a hint that the post is going to end up positive, otherwise people might only read a paragraph or two and assume it's a eulogy. (I know the title is "Reviving Sandstorm", but I think some people won't get the hint from that alone.)

Reply:
Done.